MiaServiceDotNetLibrary.Tests/MiaEnvsConfigurationsImpl.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using MiaServiceDotNetLibrary.Environment;
+namespace MiaServiceDotNetLibrary.Tests
+{
+    public class MiaEnvsConfigurationsImpl: MiaEnvsConfigurations
```

Comment:
A developer using the lib will extend this class or the `MiaEnvsConfigurations`? 

If the answer is the second, what is the purpose of this class then?

Reply:
Yes, this is a test class, I use it into the tests only, because the `MiaEnvsConfigurations` class is an abstract one